mtgjson5/providers/scryfall.py

```diff
@@ -29,6 +29,25 @@ class ScryfallProvider(AbstractProvider):
     CARDS_IN_SET: str = "https://api.scryfall.com/cards/search?order=set&q=set:{0}%20unique:prints"
     TYPE_CATALOG: str = "https://api.scryfall.com/catalog/{0}"
     cards_without_limits: Set[str]
+    language_map: Dict[str, str] = {
```

Comment:
This is defined in consts.py as `LANGUAGE_MAP` already. No need to re implement.